### DISCUSSION TITLE
Updates button - no driver or software updates listed - 2021 TUF gaming F17 FX706HC

### DISCUSSION THREAD

**Discussion by Draebor** [seerge/g-helper#552] [Q&A] [answered]
Running the latest 0.80.0.0 preview version of g helper

updates button shows bios updates but nothing under driver and software updates?

not sure if this model is supported or not for this feature yet

**Comment by seerge** [comment]
@Draebor can you post a log from %appdata%/g-helper here ?

**Reply by seerge** [reply, marked as answer]
And try this build, it should show a list for your case 

[GHelper.zip](https://github.com/seerge/g-helper/files/11676616/GHelper.zip)

**Reply by Draebor** [reply]
mark as done,  driver update list showing on this build. thanks !
